i18n/zh/docusaurus-plugin-content-docs/current/architecture/cloud/edge_controller.md

```diff
@@ -3,57 +3,56 @@ title: Edge Controller
+## Downstream Controller：
+### 同步添加/更新/删除事件到边缘
 
-
-## Downstream Controller:
-### Sync add/update/delete event to edge
-
-- Downstream controller: Watches K8S Api-server and sends updates to edgecore via cloudHub
-- Sync (pod, configmap, secret) add/update/delete event to edge via cloudHub
-- Creates Respective manager (pod, configmap, secret) for handling events by calling manager interface
-- Locates configmap and secret should be send to which node
+- Downstream Controller：监视K8S Api-server并通过cloudHub发送更新到EdgeCore
+- 通过cloudHub同步Pod,ConfigMap,Secret）的添加/更新/删除事件到边缘
+- 通过调用管Upstream Controller理接口创建相应的管理器（Pod,ConfigMap,Secret）来处理事件
+- 定位ConfigMap和Secret应发送到哪个节点
 
 ![Downstream Controller](/img/edgecontroller/DownstreamController.png)
-## Upstream Controller:
-### Sync watch and update status of resource and events
-
-- UpstreamController receives messages from edgecore and syncs the updates to K8S-Api-server
-- Creates stop channel to dispatch and stop event to handle pods, configMaps, node and secrets
-- Creates message channel to update Nodestatus, Podstatus, Secret and configmap related events
-- Gets pod condition information like Ready, Initialized, Podscheduled and Unschedulable details
-- **Below is the information for PodCondition**
-   - **Ready**: PodReady means the pod is able to service requests and should be added to the load balancing pools for all matching services
-   - **PodScheduled**: It represents the status of the scheduling process for this pod
-   - **Unschedulable**: It means the scheduler cannot schedule the pod right now, maybe due to insufficient resources in the cluster
-   - **Initialized**: It means that all Init containers in the pod have started successfully
-   - **ContainersReady**: It indicates whether all containers in the pod are ready
-- **Below is the information for PodStatus**
-   - **PodPhase**: Current condition of the pod
-   - **Conditions**: Details indicating why the pod is in this condition
-   - **HostIP**: IP address of the host to which pod is assigned
-   - **PodIp**: IP address allocated to the Pod
-   - **QosClass**: Assigned to the pod based on resource requirement
+
+## Upstream Controller：
+### 同步监视并更新资源和事件的状态
+
+- Upstream Controller接收来自EdgeCore的消息，并将更新同步到K8S Api-server
+- 创建停止通道以分发和停止处理Pod,ConfigMap节点和Secret事件
+- 创建消息通道以更新NodeStatus,PodStatus,Secret和ConfigMap相关事件
+- 获取Pod状态信息，如Ready,Initialized,PodScheduled和Unschedulable详细信息
+- **以下是 PodCondition 的信息**
+   - **Ready**：PodReady表示Pod能够提供服务请求，并应添加到所有匹配服务的负载均衡池中
+   - **PodScheduled**：它表示此Pod的调度过程状态
+   - **Unschedulable**：表示调度器当前无法调度此Pod，可能是由于集群中的资源不足
+   - **Initialized**：表示Pod中的所有初始化容器均已成功启动
+   - **ContainersReady**：表示Pod中的所有容器是否都已准备好
+- **以下是 PodStatus 的信息**
+   - **PodPhase**：Pod的当前状态
+   - **Conditions**：指示Pod处于此状态的详细信息
+   - **HostIP**：分配给Pod的主机的IP地址
+   - **PodIp**：分配给Pod的IP地址
+   - **QosClass**：根据资源需求分配给Pod的QoS类别
```

Comment:
![image](https://github.com/user-attachments/assets/8f8c1c58-24e3-4a78-91e8-2c7e14ed4989)
It would be better if the format here is adjusted according to paragraphs.

Reply:
![image](https://github.com/user-attachments/assets/7cdd9a32-3bc3-4e55-8874-dfb8a2c4a48b)
Updated

Reply:
![image](https://github.com/user-attachments/assets/c73647d8-24ae-45da-8e32-fb8cd4de8611)
Updated to adjust the format